emukit/test_functions/quadrature/baselines.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+from typing import Tuple, List
+import numpy as np
+from scipy.integrate import quad, dblquad
+
+
+def univariate_approximate_ground_truth_integral(func, integral_bounds: Tuple[float, float]):
+    """
+    Uses scipy.integrate.quad to estimate the ground truth integral
```

Comment:
Minor: this gives out implementation detail. Better phrasing would be "Estimate the ground truth integral." But for test functions this is very minor concern

Reply:
sure, I'll rephrase.